app/pages/collections/list.cjsx

```diff
@@ -64,24 +132,40 @@ List = React.createClass
             </div>
             <nav>
               {if meta
+                buttonClasses = classNames {
+                  "pill-button": true
+                  "project-pill-button": @props.project?
+                }
                 <nav className="pagination">
                   {for page in [1..meta.page_count]
                     active = (page is +location.query.page) or (page is 1 and not location.search)
                     <Link
-                      key={page}
-                      to={"#{@props.location.pathname}?page=#{page}"}
-                      activeClassName="active"
-                      className="pill-button"
-                      style={border: "2px solid" if active}>
+                      key = {page}
```

Comment:
I guess that contradicts this advice: https://github.com/zooniverse/Panoptes-Front-End/pull/2804#discussion_r71548262

When do we have spaces and when don't we? spaces for assignments but not for attributes, is that it?